setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(name = 'pytest-ignore-flaky',
       description = 'ignore failures from flaky tests (pytest plugin)',
-      version = '0.1.1',
+      version = '0.1.2',
```

Comment:
Since this requires a major upgrade on py.test, we also need to bump our version number to `0.2.0`.
Or lets call it `1.0.0` :smile:

Reply:
Sounds good, version changed.